src/metabase/integrations/slack.clj

```diff
@@ -1,133 +1,107 @@
 (ns metabase.integrations.slack
-  (:require [cheshire.core :as cheshire]
+  (:require [clojure.tools.logging :as log]
+            [cheshire.core :as json]
             [clj-http.client :as http]
-            [clojure.tools.logging :as log]
-            [metabase.models.setting :refer [defsetting]]))
+            [metabase.models.setting :refer [defsetting]]
+            [metabase.util :as u]))
 
 
 ;; Define a setting which captures our Slack api token
 (defsetting slack-token "Slack API bearer token obtained from https://api.slack.com/web#authentication")
 
-(def ^:private ^:const slack-api-baseurl "https://slack.com/api")
-(def ^:private ^:const metabase-slack-files-channel "metabase_files")
+(def ^:private ^:const ^String slack-api-base-url "https://slack.com/api")
+(def ^:private ^:const ^String files-channel-name "metabase_files")
 
+(def ^{:arglists '([])} slack-configured?
+  "Is Slack integration configured?"
+  (comp boolean seq slack-token))
```

Comment:
For making things extra 1337 yo!

But real talk I thought the same thing when I got started today and this is already switched back to a regular function